tests/test_autogen_diffs.py

```diff
@@ -39,9 +47,10 @@
 from alembic.testing import is_not_
 from alembic.testing import mock
 from alembic.testing import TestBase
+from alembic.testing.env import clear_staging_env, staging_env
 from alembic.util import CommandError
-from ._autogen_fixtures import AutogenFixtureTest
-from ._autogen_fixtures import AutogenTest
+from ._autogen_fixtures import _default_object_filters, AutogenFixtureTest, \
```

Comment:
**mike bayer** (zzzeek) wrote:

this is all going to reformat w/ black...if you install the .pre-commit-config.yml hook for your next commit

Reply:
is the project switching to black? I had it in black format and undid it in some places to match the older style haha. I approve! will do